src/libraries/Common/src/Interop/Interop.ICU.cs

```diff
@@ -11,6 +12,9 @@ internal static partial class Globalization
         [DllImport(Libraries.GlobalizationNative, EntryPoint = "GlobalizationNative_LoadICU")]
         internal static extern int LoadICU();
 
+        [DllImport(Libraries.GlobalizationNative, EntryPoint = "GlobalizationNative_InitICUFunctions")]
```

Comment:
>GlobalizationNative_InitICUFunctions [](start = 64, length = 36)

why we are using different name InitICUFunctions than InitICU is the method header?

Reply:
I think just because I suck at naming. Will update them to match.